lib/run/index.js

```diff
@@ -71,6 +71,14 @@ module.exports = function (options, callback) {
         if (!_.isObject(options.collection)) {
             return callback(new Error('newman: expecting a collection to run'));
         }
+        // sanitize environment option
+        if (!options.environment) {
```

Comment:
I'm sorry but I think I wasn't that clear. I meant the config inside run. @czardoz can you look into it?